test-prow-e2e.sh

```diff
@@ -49,4 +49,6 @@ elif [ "$SCENARIO" == "olmFull" ]; then
   ./test-cypress.sh -p olm -h true
 elif [ "$SCENARIO" == "ceph" ]; then
   ./test-cypress.sh -p ceph -h true
+elif [ "$SCENARIO" == "kubevirt-gating" ]; then
+  ./test-cypress.sh -p kubevirt -h true
```

Comment:
I assume you want to skip the Protractor tests as well (line 37 above).

Reply:
No, we still want to execute proctator tests too. We will remove them once we rewrite protractor gating suite to cypress.